sensor_msgs/Image input_image
---
std_msgs/String objects_json